tests/unit/test_lib/test_utilities/test_filters/test_windowing_scheme.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
```

Comment:
We should follow the same directory structure. i.e., `test_filters/test_windowing_scheme/test_windowing_scheme.py`